db/mysql/schema.sql

```diff
@@ -306,6 +355,12 @@ CREATE TABLE `usr_user_role`
   PRIMARY KEY (`user_id`, `role_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `usr_extuser` (
```

Comment:
it is still valid, however in our case internal and external user ids are
equal - problem is that there is no other cas attribute which must be
unique (and can be used as uv internal user id) - only cas userid

On Fri, Mar 20, 2015 at 8:53 PM, Tomas Knap <notifications@github.com>
wrote:

> In db/mysql/schema.sql
> <https://github.com/UnifiedViews/Core/pull/282#discussion_r26870878>:
>
> > @@ -306,6 +355,12 @@ CREATE TABLE `usr_user_role`
> >    PRIMARY KEY (`user_id`, `role_id`)
> >  ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
> >
> > +CREATE TABLE `usr_extuser` (
>
> You are writing "original purpose". So is this purpose still valid?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UnifiedViews/Core/pull/282/files#r26870878>.
>